src/out_hoi4/countries/out_country.cpp

```diff
@@ -39,6 +39,58 @@ void OutputPolitics(std::ostream& country_history,
    country_history << "}\n";
    country_history << "\n";
 }
+void OutputPuppets(std::ostream& output,
+    const std::string& tag,
+    const std::string& governmentIdeology,
```

Comment:
Change governmentIdeology to government_ideology and then fix everything in the function